src/dune

```diff
@@ -1,8 +1,8 @@
-(ocamllex prelexer)
+(ocamllex prelexer REBracketExpressionLexer)
 
 (menhir
  (flags --explain --inspection --table)
- (modules parser))
+ (modules parser REBracketExpressionParser))
```

Comment:
Btw, do we need all these flags for the parser?

Reply:
I mean, I just kept them as they were in the Makefile before, but I'm not sure about what they do.

Reply:
Yes, we need them. `--inspection` is important to the incremental parser engine. `--explain` is to get `.conflicts` files. `--table` is to use the interpreter back-end.

Reply:
Aren't `.conflict` files meant more for debugging?

Reply:
Yes, but they are not really expansive to produce. But, OK, this can moved to the debugging mode of the build system, right.